src/redux/missions/missionsSlice.js

```diff
@@ -21,7 +22,14 @@ const fetchMissions = createAsyncThunk('missions/fetchMissions', async () => {
 const missionsSlice = createSlice({
   name: 'missions',
   initialState,
-  reducers: {},
+  reducers: {
+    member: (state, action) => {
+      const missionId = action.payload;
+      state.missions = state.missions.map((mission) => (mission.mission_id === missionId
+        ? { ...mission, activeMember: !mission?.activeMember ?? true }
+        : mission));
+    },
+  },
```

Comment:
You did great with this logic, the use of the terenary function ( ? : ) is great! good job 😄
Please help me change the key from `activeMember` to `reserved`

It's ok that you use this name in all other instances but the Requirements ask for the key `reserved`

Reply:
Done. thank you @Estete9 👍😄